### PR TITLE
Separate Live and Test schemas in the dropdown list

### PR DESCRIPTION
**Pull request by jonnyshaw89**
Split the dropdown list with option group headings

### CODE REVIEW

**Review by nealedj — CHANGES_REQUESTED** [review]
LGTM - just linting

File: surveys/surveys.go [hunk 1]
```diff
@@ -54,134 +60,140 @@ func LauncherSchemaFromFilename(filename string) LauncherSchema {
...
-		LauncherSchemaFromFilename("test_view_submitted_response.json"),
-		LauncherSchemaFromFilename("test_textarea.json"),
-		LauncherSchemaFromFilename("test_textfield.json"),
-		LauncherSchemaFromFilename("test_timeout.json"),
-		LauncherSchemaFromFilename("test_total_breakdown.json"),
-		LauncherSchemaFromFilename("test_unit_patterns.json"),
+func GetAvailableSchemas() LauncherSchemas {
+	schemaList := LauncherSchemas{
+		Live: []LauncherSchema{
+			LauncherSchemaFromFilename("0_star_wars.json"),
```
Comment (dwyeradam) [inline surveys/surveys.go]: Can we go through as say what is actually live before merging. For example Star Wars isn't surely.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: surveys/surveys.go (hunk 1 above)
Comment (jonnyshaw89) [reply surveys/surveys.go]: The `Live` and `Test` categories are a bit confusing? we could have a schema that isnt a test schema, but isn't `Live` as the business would call it.
But Star Wars should be in the other list

Comment (shahi645) [reply surveys/surveys.go]: I thought we always wanted 3 categories, Business/Social/Test

File: surveys/surveys.go [hunk 2]
```diff
@@ -99,10 +98,11 @@ func GetAvailableSchemas() LauncherSchemas {
 			LauncherSchemaFromFilename("mbs_0867.json"),
 			LauncherSchemaFromFilename("mbs_0873.json"),
 			LauncherSchemaFromFilename("mci_transformation.json"),
```
Comment (shahi645) [inline surveys/surveys.go]: Technically these aren't live yet, I was thinking, should have 3 drop downs...i.e `Business`/`Social`/`Test`

Comment (jonnyshaw89) [reply surveys/surveys.go]: This wasn't recorded in the Trello card. When was this decided?

Comment (shahi645) [reply surveys/surveys.go]: we talked about it at a tech session, but you're right should have been on the ticket

Comment (dwyeradam) [reply surveys/surveys.go]: This is however on the card.
`Split based on **{group}_{type}** format`
So we should really have mbs_, census_, e_commerce, lms_, mci_, rsi_ on their own. No point splitting 1_ and 2_ though.
mci/rsi_tranformation could be grouped into a Transformations group.

**Comment by jonnyshaw89** [comment]
Im swaying towards `Business/Social/Test`.
Any objections?

**Comment by nealedj** [comment]
If we're splitting on those lines then probably keep census in its own group.

**Review by nealedj — CHANGES_REQUESTED** [review]
Nice - just needs changes for the linter.

**Comment by shahi645** [comment]
good to approve when linting passes

**Comment by jonnyshaw89** [comment]
@dwyeradam @nealedj I have fixed the linting and squashed.

**Review by dwyeradam — APPROVED** [review]

**Review by nealedj — APPROVED** [review]

**Pull request merged**